### DISCUSSION TITLE
dnscrypt not resolving->nslookup->I only get dnscrypt blocked AAAA queries (no ipv6)[expected]

### DISCUSSION THREAD

**Discussion by Romulo-Romero** [DNSCrypt/dnscrypt-proxy#1734] [Q&A]
Hi,
I have been trying to debug a specific url in order to find out if there is some sort of URL blocking here on my side, which is not the case, since I was able to access in this particular case the website using other means instead of using dnscrypt, and of course the "rcode" didn't return the expected behavior as in a url blocking.

But I couldn't figure out what could be wrong with my dnscrypt config. 
Maybe this: 
HINFO CPU = This query has been locally blocked OS = by dnscrypt-proxy
        ttl = 1 (1 sec)

So, here it is the full trace of nslookup:
`------------
Got answer:
    HEADER:
        opcode = QUERY, id = 2, rcode = NOERROR
        header flags:  response, recursion avail.
        questions = 1,  answers = 1,  authority records = 0,  additional = 1

    QUESTIONS:
        somewebsite.com, type = A, class = IN
    ANSWERS:
    ->  somewebsite.com
        HINFO CPU = This query has been locally blocked OS = by dnscrypt-proxy
        ttl = 1 (1 sec)
    ADDITIONAL RECORDS:
    ->  (root)
        ??? unknown type 41 ???
        ttl = 0 (0 secs)

------------
Non-authoritative answer:
------------
Got answer:
    HEADER:
        opcode = QUERY, id = 3, rcode = NOERROR
        header flags:  response, recursion avail.
        questions = 1,  answers = 1,  authority records = 1,  additional = 0

    QUESTIONS:
        somewebsite.com, type = AAAA, class = IN
    ANSWERS:
    ->  somewebsite.com
        HINFO CPU = AAAA queries have been locally blocked by dnscrypt-proxy    OS = Set block_ipv6 to false to disable this feature
        ttl = 86400 (1 day)
    AUTHORITY RECORDS:
    ->  gov.br
        ttl = 60 (1 min)
        primary name server = a.root-servers.net
        responsible mail addr = h.invalid
        serial  = 1
        refresh = 10000 (2 hours 46 mins 40 secs)
        retry   = 300 (5 mins)
        expire  = 604800 (7 days)
        default TTL = 2400 (40 mins)

------------
------------
Got answer:
    HEADER:
        opcode = QUERY, id = 4, rcode = NOERROR
        header flags:  response, recursion avail.
        questions = 1,  answers = 1,  authority records = 0,  additional = 1

    QUESTIONS:
        somewebsite.com, type = A, class = IN
    ANSWERS:
    ->  somewebsite.com
        HINFO CPU = This query has been locally blocked OS = by dnscrypt-proxy
        ttl = 1 (1 sec)
    ADDITIONAL RECORDS:
    ->  (root)
        ??? unknown type 41 ???
        ttl = 0 (0 secs)

------------
Non-authoritative answer:
------------
Got answer:
    HEADER:
        opcode = QUERY, id = 5, rcode = NOERROR
        header flags:  response, recursion avail.
        questions = 1,  answers = 1,  authority records = 1,  additional = 0

    QUESTIONS:
        somewebsite.com, type = AAAA, class = IN
    ANSWERS:
    ->  somewebsite.com
        HINFO CPU = AAAA queries have been locally blocked by dnscrypt-proxy    OS = Set block_ipv6 to false to disable this feature
        ttl = 86400 (1 day)
    AUTHORITY RECORDS:
    ->  gov.br
        ttl = 60 (1 min)
        primary name server = a.root-servers.net
        responsible mail addr = h.invalid
        serial  = 1
        refresh = 10000 (2 hours 46 mins 40 secs)
        retry   = 300 (5 mins)
        expire  = 604800 (7 days)
        default TTL = 2400 (40 mins)

------------
Name:    somewebsite.com`
Thanks

**Comment by jedisct1** [comment]
>   `AAAA queries have been locally blocked by dnscrypt-proxy    OS = Set block_ipv6 to false to disable this feature`

This is a request for an IPv6 address, but you're blocking IPv6. Set block_ipv6 to false to disable this feature.

**Reply by Romulo-Romero** [reply]
Stupid me, I had forgotten to edit my blocked-ips.txt file, and never bothered checking the log file (Checked nxdomain and debug log except this one ) -.-

Anyways, i didn't think it would have anything to do with the AAAA queries, since it makes no sense to me, I am using ipv4 and it wouldn't make sense trying yo resolve addresses that leads to ipv6. At least as far as I can tell.
